manifests/m/mtkennerly/madamiru/0.1.0/mtkennerly.madamiru.installer.yaml

```diff
@@ -0,0 +1,42 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: mtkennerly.madamiru
+PackageVersion: 0.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: madamiru.exe
+  PortableCommandAlias: madamiru
+FileExtensions:
+- avi
+- bmp
+- flac
+- gif
+- ico
+- jpeg
+- m4a
+- m4v
+- madamiru
+- mkv
+- mov
+- mp3
+- mp4
+- png
+- svg
+- tiff
+- wav
+- webm
+- webp
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: gstreamerproject.gstreamer
```

Comment:
```suggestion
  - PackageIdentifier: gstreamerproject.gstreamer
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
Done in b06ab50d472427056466db13666ae080a972ab16 :) I added it at the installer level since VCRedist has x64/x86 versions.